lms/djangoapps/courseware/views/views.py

```diff
@@ -994,6 +1008,10 @@ def course_about(request, course_id):
             # context. This value is therefor explicitly set to render the appropriate header.
             'disable_courseware_header': True,
             'pre_requisite_courses': pre_requisite_courses,
+            'course_requirements': course_requirements if course_requirements else None,
```

Comment:
course_requirements_for_courseware_views returns {} in case things don't go as expected, so you don't need to return None. @anfbermudezme